stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/User.kt

```diff
@@ -63,4 +63,16 @@ public data class User(
     @IgnoreDeserialisation
     override var extraData: MutableMap<String, Any> = mutableMapOf()
 
-) : CustomObject
+) : CustomObject {
+    var name: String
+        get() = getExternalField(this, EXTRA_NAME)
+        set(value) {
+            extraData[EXTRA_NAME] = value
+        }
+
+    var image: String
```

Comment:
Have we created a followup task to make these real properties and move them into the constructor? (That will also remove the values from extraData, which is very breaking)

Reply:
I'm not sure we will be able to remove it completely, check this [thread](https://github.com/GetStream/stream-chat-android/pull/2278#discussion_r696788612)

Reply:
We _can_ remove it, it will just break the code of anyone accessing these values through `extraData`, so we'll have to do it in a major version or something

Reply:
Thanks for reminding me!